src/net_processing.cpp

```diff
@@ -1115,6 +1123,7 @@ PeerLogicValidation::PeerLogicValidation(CConnman* connmanIn, BanMan* banman, CS
 {
     // Initialize global variables that cannot be constructed at startup.
     recentRejects.reset(new CRollingBloomFilter(120000, 0.000001));
+    recent_confirmed_transactions.reset(new CRollingBloomFilter(24000, 0.000001));
```

Comment:
should the parameters to the bloom filter be tunnable with commandline args?

Reply:
I can't imagine why anyone would need to customize this parameter (it's not like we offered the ability to customize how many outputs we'd check in the cache, or the ability to customize the recentRejects filter).